account_move_line_product/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=========================
+Account Move Line Product
+=========================
+
+This module enhances the views associated to account moves and account move
+lines to display the product that is associated to the move line.
+
+This will be relevant in perpetual inventory, for inventory-related account
```

Comment:
@jbeficent Ok for that context in particular, but as this module is just dependent of 'account' module and does not involve stock, I will tend to put in account-financial-tools.